test/bin/find.js

```diff
@@ -80,32 +89,47 @@ describe('bin', function() {
     process.argv[2] = '-u'
     proxyquire('../../src/bin/find', stub)
     assert.ok(getUnusedRules.called)
+    process.argv[2] = '--unused'
+    proxyquire('../../src/bin/find', stub)
+    assert.ok(getUnusedRules.called)
   })
 
-  it('options -u|--unused and no unused rules found', function() {
+  it('options -u|-unused and no unused rules found', function() {
     getUnusedRules.returns([])
     process.exit = function(status) {
       assert.equal(status, 0)
     }
     process.argv[2] = '-u'
     proxyquire('../../src/bin/find', stub)
     assert.ok(getUnusedRules.called)
+    process.argv[2] = '--unused'
+    proxyquire('../../src/bin/find', stub)
+    assert.ok(getUnusedRules.called)
   })
 
-  it('option -u|--unused along with -n|no-error', function() {
+  it('option -u|-unused along with -n|-no-error', function() {
     process.exit = function(status) {
       assert.equal(status, 0)
     }
     process.argv[2] = '-u'
     process.argv[3] = '-n'
     proxyquire('../../src/bin/find', stub)
     assert.ok(getUnusedRules.called)
+
+    process.argv[2] = '--unused'
+    process.argv[3] = '-no-error'
```

Comment:
Seems to me it's a bug in `yargs`, when it's `--used` (double dash), but `-no-error` (single dash); shouldn't that be further looked into and rather fixed upstream (in `yargs`, should the bug really be located there)?